packages/theme/components/MyAccount/SavedAddressForm.vue

```diff
@@ -53,10 +53,11 @@
         />
       </ValidationProvider>
       <SfInput
-        data-cy="shipping-details-input_apartment"
+        data-cy="savedAddress-details-input_apartment"
         v-model="form.addressLine2"
         name="apartment"
         label="House/Apartment number"
+        required
```

Comment:
We've discussed that `addressLine2` should be optional. (https://github.com/vuestorefront/spree/issues/114)

Reply:
Fixed.